docs/en/how-to/deploy_toolbox_docker.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Deploy to Docker"
+type: docs
+weight: 2
+description: >
+  How to set up and configure Toolbox to run on Docker.
+---
+
+
+## Before you begin
+
+1. [Install](https://docs.docker.com/engine/install/) the Docker engine.
```

Comment:
I think this would need to be Docker Compose, and not just docker?
```suggestion
1. [Install Docker Compose.](https://docs.docker.com/compose/install/)
```